robottelo/api/utils.py

```diff
@@ -968,3 +968,30 @@ def set_hammer_api_timeout(timeout=-1, reverse=False):
                 new_timeout, default_timeout
             )
         )
+
+
+def update_rhsso_settings_in_satellite(revert=False):
```

Comment:
Consider changing how this function works, to make it more usable with tests and parametrization.  If the function took a dictionary of settings, then tests could actually use this function to change settings relevant to test data or cases - instead of just statically configuring whatever is in the properties file.

You can update the passed dictionary with values from the properties files for anything that wasn't passed in, retaining your default behavior here.

Reply:
If you think strictly this needs to be done in this PR  I will do it. As this is old code, I will have te refactor everything can take more time. If this PR has already over more than 30 days. Based on what you have suggested can be done as part of another issue, unless you do not agree.

Reply:
Ah, I see now this was just moved out of a test module. Agreed this could be enhanced later, since you didn't write it specifically for this PR.

Reply:
create issue here https://github.com/SatelliteQE/robottelo/issues/7990 will work on future